deploy/olm-catalog/eunomia/eunomia.package.yaml

```diff
@@ -0,0 +1,4 @@
+packageName: eunomia
```

Comment:
Same here ... why do we need this file in our repo? The file is in the https://github.com/operator-framework/community-operators repo.